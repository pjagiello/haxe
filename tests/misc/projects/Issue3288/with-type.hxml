--display MainWithType.hx@59
