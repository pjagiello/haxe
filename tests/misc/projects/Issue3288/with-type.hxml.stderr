<list>
<i n="AnotherOne"><t>AnotherOne</t><d></d></i>
<i n="OtherType"><t>OtherType</t><d></d></i>
<i n="f1"><t>Int</t><d></d></i>
<i n="f2"><t>Void -&gt; Void</t><d></d></i>
</list>

